### DISCUSSION TITLE
📣 제페토 월드 공모전 당선자 발표 일정 안내

### DISCUSSION THREAD

**Discussion by Zepeto-TechEvangelist** [naverz/zepeto-studio-kor#982] [1. Announcements]
안녕하세요~ 제페토 에반젤리스트입니다.

제페토 월드 공모전 작품은 현재 심사 중입니다! 🌏
당선작 안내는 7/25 (월)에 진행 예정이에요.
과연 총 상금 2천 만원의 주인공은 누구일지 조금만 더 기다려주세요!🤗

👉 [월드 공모전 내용 복습하기](https://zefro.zepeto.me/nds/event/T3sxp3wK88AsY9WuXiyx4rU?lang=ko)

![SNS_220711_event_월드 공모전_심사일정 발표](https://user-images.githubusercontent.com/100118650/178174757-d7675c69-d0b9-4213-b928-87f8208ee5b3.png)

**Comment by Chordcode002** [comment]
25일 이전까지 추가된 기능이나 수정사안도 심사에 반영이 되나요? 아니면 현재까지 업로드된 버전으로만 심사가 되는건가요?

**Reply by Zepeto-TechEvangelist** [reply]
안녕하세요, 현재까지 업로드된 버전으로만 심사가 됩니다.
